exercises/basic_tunnel/basic_tunnel.p4

```diff
@@ -87,7 +87,24 @@ parser MyParser(packet_in packet,
 *************************************************************************/
 
 control MyVerifyChecksum(inout headers hdr, inout metadata meta) {
-    apply {  }
+    apply { 
+        /*
+        Control function for checksum verification.
+       
+        This function is responsible for the MyVerifyChecksum control block in P4, 
+        which checks and drops IPv4 packets with incorrect checksums. It uses inout
+        headers hdr to manipulate packet headers and inout metadata meta to store 
```

Comment:
This block is responsible for checking checksums in input packet headers, but it is _not_ responsible for dropping packets with incorrect checksums.  See https://github.com/p4lang/behavioral-model/blob/main/docs/simple_switch.md#restrictions-on-code-in-the-verifychecksum-control for details.

Reply:
Okay, I fixed it. By the way, then which block takes care of dropping packets with incorrect checksums?